### PR TITLE
DON-950: Add regression test check for server side rendering

### PR DESCRIPTION
**Pull request by bdsl**
Other tests are failing for unrelated reasons, but this new one is passing:

```
[chrome 117.0.5938.89 windows #0-4] Running: chrome (v117.0.5938.89) on windows
[chrome 117.0.5938.89 windows #0-4] Session ID: 1b9b1fa08735-5829bf74be9b-be36fce7ae6e-171222908495-16840990
[chrome 117.0.5938.89 windows #0-4]
[chrome 117.0.5938.89 windows #0-4] » /features/server-side-rendering.feature
[chrome 117.0.5938.89 windows #0-4] The site is rendered server side
[chrome 117.0.5938.89 windows #0-4] We use server side rendering to make the site faster to use for people and easier for search engines to index.
[chrome 117.0.5938.89 windows #0-4]
[chrome 117.0.5938.89 windows #0-4] Homepage is server side rendered
[chrome 117.0.5938.89 windows #0-4]    ✓ Given I am loading the site without Javascript
[chrome 117.0.5938.89 windows #0-4]    ✓ When I view the homepage
[chrome 117.0.5938.89 windows #0-4]    ✓ Then I should see in the source "Double the difference"
[chrome 117.0.5938.89 windows #0-4]
[chrome 117.0.5938.89 windows #0-4] 3 passing (833ms)
```

### CODE REVIEW

File: steps/ssr.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+Given(/^I am loading the site without Javascript$/, async () => {
+    // no-op
+});
+
+When(/^I view the homepage$/, async () => {
+    const client = axios.create({
+        baseURL: 'https://mailtrap.io',
+        headers: {
+            'Api-Token': process.env.MAILTRAP_API_TOKEN,
+        },
```
Comment (dorota-joanna) [inline steps/ssr.js:21]:
```suggestion
```

File: steps/ssr.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    });
```
Comment (dorota-joanna) [inline steps/ssr.js:22]:
```suggestion
    const client = axios.create();
```

**Review by dorota-joanna — APPROVED** [review]
LGTM

**Review by NoelLH — APPROVED** [review]
Nice. We in theory use a health ping tool to confirm this on one Donate Regression page too, but certainly good to have it validated in an extra place

**Pull request merged**